bitnami/redis-cluster/Chart.yaml

```
@@ -23,4 +23,4 @@ name: redis-cluster
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/redis
   - http://redis.io/
-version: 8.3.5
+version: 8.3.6
```

Thanks for your contribution, but can you upgrade the version? This is a new feature, it should be a minor version.
```suggestion
version: 8.4.0
```